warehouse/manage/views.py

```diff
@@ -325,6 +325,12 @@ def default_response(self):
 
     @view_config(route_name="manage.account.totp-provision.image", request_method="GET")
     def generate_totp_qr(self):
+        if not self.request.user.two_factor_provisioning_allowed:
+            self.request.session.flash(
+                "Modifying 2FA requires a verified email.", queue="error"
```

Comment:
This message (and other instances of it) aren't quite right: As long as we have `two_factor_prohibited`, this might fail even if the user has a valid email.

Reply:
Yep, I'll just remove `two_factor_prohibited` from the PR entirely. It was there to allow us to feature-gate on test.pypi.org, but since we're doing the full rollout it doesn't serve a purpose anymore 😄